lib/MadNLPGPU/src/MadNLPGPU.jl

```diff
@@ -18,8 +18,9 @@ import MadNLP:
     AbstractOptions, AbstractLinearSolver, AbstractNLPModel, set_options!,
     SymbolicException,FactorizationException,SolveException,InertiaException,
     introduce, factorize!, solve!, improve!, is_inertia, inertia, tril_to_full!,
-    LapackOptions, input_type, is_supported
+    LapackOptions, input_type, is_supported, symul!
 
+symul!(y, A, x::CuVector{T}, α = 1., β = 0.) where T = CUBLAS.symv!('L', T(α), A, x, T(β), y)
```

Comment:
Maybe add a comment to explain why we need to implement a `symul!` function wrapping `symv`?